cupyx/scipy/sparse/sputils.py

```diff
@@ -1,5 +1,13 @@
 import cupy
 
+from cupy.core._dtype import get_dtype
```

Comment:
According to the OpenStack guideline, please only import modules.
```suggestion
from cupy.core import _dtype
```